CHANGELOG.md

```diff
@@ -54,6 +54,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed ORT bug miscalculating Mid Max Origin Connections as all servers, usually resulting in 1.
 - Added Delivery Service Raw Remap `__RANGE_DIRECTIVE__` directive to allow inserting the Range Directive after the Raw Remap text. This allows Raw Remaps which manipulate the Range.
 - Added an option for `coordinateRange` in the RGB configuration file, so that in case a client doesn't have a postal code, we can still determine if it should be allowed or not, based on whether or not the latitude/ longitude of the client falls within the supplied ranges. [Related github issue](https://github.com/apache/trafficcontrol/issues/4372)
+- Fixed #3548 - Setting order in Portal can create failing CR-config
```

Comment:
Can you make this a little more descriptive. maybe just use the PR title? `Prevents DS regexes with non-consecutive order from generating invalid CRconfig`